chall07/sluhtala.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+import sys
+
+def put_usage():
+    usage = "usage: ./sluhtala.py <1-9 squared_rows...>"
+    print(usage)
+    exit()
```

Comment:
don't hardcode your login use sys.argv[0] instead.

Plus you can use exit argument to avoid using print.